src/extensions/addrbar/bootstrap-table-addrbar.js

```diff
@@ -86,25 +105,26 @@ $.BootstrapTable = class extends $.BootstrapTable {
 
       const _prefix = this.options.addrPrefix || ''
       const _onLoadSuccess = this.options.onLoadSuccess
+      const _onPageChange = this.options.onPageChange
 
       this.options.onLoadSuccess = data => {
         if (this.addrbarInit) {
           this.addrbarInit = false
         } else {
-          const params = {}
-          params[`${_prefix}page`] = this.options.pageNumber,
-          params[`${_prefix}size`] = this.options.pageSize,
-          params[`${_prefix}order`] = this.options.sortOrder,
-          params[`${_prefix}sort`] = this.options.sortName,
-          params[`${_prefix}search`] = this.options.searchText
-          // h5提供的修改浏览器地址栏的方法
-          window.history.pushState({}, '', _buildUrl(params))
+          _updateHistoryState(this,_prefix)
         }
 
         if (_onLoadSuccess) {
           _onLoadSuccess.call(this, data)
         }
       }
+      this.options.onPageChange = (number,size) => {
+        _updateHistoryState(this,_prefix)
+
+        if (_onPageChange) {
+          _onPageChange.call(this,number,size)
```

Comment:
I think we can use apply in order to send all arguments.

Something like yourFunction.apply(...args)

Reply:
I do it, like in L104-106.

Reply:
Ok, I changed this.